### PR TITLE
Add callback to end of temperature conversion.

### PR DESCRIPTION
**Pull request by Ogyb**
void StartMeasure();` --Only added spaces 2 comment (Git edit = del -> add) i no delete any function. 

This library working correctly on my device ~8 month.

I used 2 temperature sensor send to Thingspeak every 30s.

Used:

```
void sendData()
{
     /*New read temperature data processing*/
}

ReadTemp.RegisterEndCallback(sendData);
```
Edit or add new Sample to DS18S20 library? 

Sorry - i little problem with using GitHub;) The first time I use GitHub, but I think I understand how it works....

### CODE REVIEW

File: Sming/Libraries/DS18S20/ds18s20.h [hunk 1]
```diff
@@ -54,6 +56,7 @@ class DS18S20
 	uint8_t numberOf=0;
 	uint8_t numberOfread=0;
 
+	void (*ReadEndCallback)() = 0;
```
Comment (slaff) [inline Sming/Libraries/DS18S20/ds18s20.h]: Please, remove this and declare a delegate.

File: Sming/Libraries/DS18S20/ds18s20.h [hunk 2]
```diff
@@ -28,7 +28,9 @@ class DS18S20
 public:
 	DS18S20();
 	void Init(uint8_t);			//call for OneWire init
-	void StartMeasure();        //Start measurement result after 1.2 seconds * number of sensors
+	void StartMeasure();                    //Start measurement result after 1.2 seconds * number of sensors
+	void RegisterEndCallback(void (*)());   //Add function called of conversion end - "interrupt" function of end conversion
```
Comment (slaff) [inline Sming/Libraries/DS18S20/ds18s20.h]: Replace the method parameter with delegate.

**Comment by Ogyb** [comment]
Ok, i read manuals and edit Callback to Delegate.

**Comment by Ogyb** [comment]
Tested and all working.

File: Sming/Libraries/DS18S20/ds18s20.h [hunk 3]
```diff
@@ -54,6 +57,7 @@ class DS18S20
 	uint8_t numberOf=0;
 	uint8_t numberOfread=0;
 
+	DS18S20CompletedDelegate ReadEndCallback = 0;
```
Comment (slaff) [inline Sming/Libraries/DS18S20/ds18s20.h]: Change the name from 'ReadEndCallback' to 'readEndCallback'.

Comment (slaff) [inline Sming/Libraries/DS18S20/ds18s20.h]: Use ' = NULL; ' instead of ' = 0; '

**Comment by Ogyb** [comment]
Ok, updated.

**Comment by slaff** [comment]
Will be merged with #884.

**Pull request closed without merging**